src/utils/accessor.ts

```diff
@@ -0,0 +1,14 @@
+import { path } from './path';
+
+export type Accessor<D, C> = string | ((row: D) => C);
```

Comment:
Maybe export it from a separate file? Cuase this is a type used in several files